### PR TITLE
GitHub Developers Portal issue templates

### PR DESCRIPTION
**Pull request by barriebyron**
hi B9Lab friends! 

The existing templates in the repo were created for specific Cosmos SDK tutorials and no longer serve the larger Developers Portal repo.

To help us capture relevant feedback and bug reports for the new (fancy!) Developers Portal, I've created new issue templates for the Developers Portal in general and updated the tutorial-specific templates.

### CODE REVIEW

**Review by nooomski — APPROVED** [review]
This looks great to me. Thanks so much @barriebyron !

File: .github/ISSUE_TEMPLATE/devportal-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: Developers Portal bug report
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-bug-report.md:2]:
```suggestion
name: Developer Portal bug report
```

File: .github/ISSUE_TEMPLATE/devportal-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+about: Create a report to help us improve
+title: ''
+labels: report
+assignees: ''
+
+---
+
+Thank you for helping us improve the Cosmos Developers Portal.
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-bug-report.md:10]:
```suggestion
Thank you for helping us improve the Cosmos Developer Portal. 
```

File: .github/ISSUE_TEMPLATE/devportal-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+about: Create a report to help us improve
+title: ''
+labels: report
+assignees: ''
+
+---
+
+Thank you for helping us improve the Cosmos Developers Portal. 
+
+Use this form to submit a bug report for the Developers Portal in general (broken link, missing content, navigation, and so on).
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-bug-report.md:12]:
```suggestion
Use this form to submit a bug report for the Developer Portal in general (broken link, missing content, navigation, and so on). 
```

File: .github/ISSUE_TEMPLATE/devportal-bug-report.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+Thank you for helping us improve the Cosmos Developers Portal. 
+
+Use this form to submit a bug report for the Developers Portal in general (broken link, missing content, navigation, and so on). 
+
+To submit a bug report for a specific tutorial, use [Cosmos SDK tutorials bug report](tutorials-bug-report.md).
+
+**What is the specific Developers Portal page?**
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-bug-report.md:16]:
```suggestion
**What is the specific Developer Portal page?**
```

File: .github/ISSUE_TEMPLATE/devportal-feature-request.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+name: Developers Portal feature request
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-feature-request.md:2]:
```suggestion
name: Developer Portal feature request
```

File: .github/ISSUE_TEMPLATE/devportal-feature-request.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+about: Suggest a feature, improvement, or idea.
+title: ''
+labels: request
+assignees: ''
+
+---
+
+Thank you for helping us improve the Cosmos Developers Portal.
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-feature-request.md:10]:
```suggestion
Thank you for helping us improve the Cosmos Developer Portal. 
```

File: .github/ISSUE_TEMPLATE/devportal-feature-request.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+about: Suggest a feature, improvement, or idea.
+title: ''
+labels: request
+assignees: ''
+
+---
+
+Thank you for helping us improve the Cosmos Developers Portal. 
+
+Use this form to submit a feature request for the Developers Portal in general.
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/devportal-feature-request.md:12]:
```suggestion
Use this form to submit a feature request for the Developer Portal in general. 
```

**Comment by coldice** [comment]
Thank you for preparing this update, content looks all good 👍 Just one minor thought: The template for tutorial changes should be applied to all tutorials, including the _b9lab-content_ (folder name to be changed soon) and the _Migrate to Starport_ content right? I was just thinking, because the existing tutorials category is specifically called _tutorials_, some users might think it only applies to these tutorials and not all. Do you think we need to explicitly state this applies to all content items, or is it clear?

File: .github/ISSUE_TEMPLATE/tutorials-bug-report.md [hunk 8]
```diff
@@ -7,20 +7,38 @@ assignees: ''
 
 ---
 
+Thank you for helping us improve Cosmos SDK tutorials. 
+
+Use this form to submit a bug report for a specific tutorial. 
+
+To submit a bug report for the Developers Portal in general, use [Developers Portal bug report](devportal-bug-report.md).
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/tutorials-bug-report.md:14]:
```suggestion
To submit a bug report for the Developer Portal in general, use [Developer Portal bug report](devportal-bug-report.md).
```

File: .github/ISSUE_TEMPLATE/tutorials-feature-request.md [hunk 9]
```diff
@@ -7,11 +7,20 @@ assignees: ''
 
 ---
 
+Thank you for helping us improve the Cosmos Developers Portal.
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/tutorials-feature-request.md:10]:
```suggestion
Thank you for helping us improve the Cosmos Developer Portal. 
```

File: .github/ISSUE_TEMPLATE/tutorials-feature-request.md [hunk 10]
```diff
@@ -7,11 +7,20 @@ assignees: ''
 
 ---
 
+Thank you for helping us improve the Cosmos Developers Portal. 
+
+Use this form to submit a feature request for a specific tutorial. 
+
+To submit a feature request for the Developers Portal in general, use [Cosmos SDK tutorials feature request](devportal-feature-request.md).
```
Comment (nooomski) [inline .github/ISSUE_TEMPLATE/tutorials-feature-request.md:14]:
```suggestion
To submit a feature request for the Developer Portal in general, use [Cosmos SDK tutorials feature request](devportal-feature-request.md).
```

**Review by nooomski — COMMENTED** [review]
Changed developerS to developer

**Review by coldice — APPROVED** [review]

**Comment by barriebyron** [comment]
Let's merge and iteratively improve... the important thing is to capture feedback. 
Next up, CODEOWNERS, see https://github.com/cosmos/sdk-tutorials/pull/864 

let's get an initial version and iterate as we go

**Pull request merged**